README.md

```diff
@@ -151,7 +151,7 @@ Add the assocated configurations in pouchd config file, e.g, `/etc/pouch/config.
 		"rune": {
             		"path": "/usr/local/bin/rune",
             		"runtimeArgs": null,
-            		"type": "io.containerd.runtime.v1.linux"
+            		"type": "io.containerd.rune.v2"
```

Comment:
pouch also supports docker API. So is it still useful to provide the info about using this type?

Reply:
I just modified the location instead of deleting the docker API.